PWGGA/GammaConv/AliAnalysisTaskGammaConvDalitzV1.h

```diff
@@ -98,7 +98,7 @@ class AliAnalysisTaskGammaConvDalitzV1: public AliAnalysisTaskSE {
     void CalculateBackground();
     void UpdateEventByEventData();
     void FillElectronQAHistos(AliAODConversionPhoton *Vgamma) const;
-    Double_t GetPsiPair( const AliDalitzAODESD *trackPos, const AliDalitzAODESD *trackNeg ) const;
```

Comment:
I don't see where you change trackPos or trackNeg, why is it needed to remove the const?